vpr/src/pack/pb_type_graph.cpp

```diff
@@ -1098,9 +1098,6 @@ static bool realloc_and_load_pb_graph_pin_ptrs_at_var(const int line_num,
         return false; //clb[9:0]123
     }
     (*token_index)++;
-    if (!checkTokenType(tokens[*token_index], TOKEN_STRING)) {
```

Comment:
Yeah, main motivation behind this is to be able to also allow numbers for pin names in pb types' ports. I think this should be allowed as if there is a multi-bit port, the bit is always enclosed in square brackets, avoiding ambiguity. I may add a `checkTokenType(..., TOKEN_DIGIT)` or similar check maybe, so that we exclude all other kinds of tokens.

Reply:
I have changed this to allow only either INT or STRING tokens